dnf-behave-tests/features/scriptlets.feature

```diff
@@ -92,3 +92,106 @@ Scenario: Remove a pkg with a failing %postun scriptlet
    And Transaction is following
        | Action        | Package                           |
        | remove        | Package-postun-fail-0:1.0-1.x86_64 |
+
+
+Scenario: Correct output for triggered successful scriptlet of package not present in transaction
+  When I execute dnf with args "install Package-triggerin-ok"
+  Then the exit code is 0
+   And Transaction is following
+       | Action        | Package                             |
+       | install       | Package-triggerin-ok-0:1.0-1.x86_64 |
+  When I execute dnf with args "install Package-install-file"
+  Then the exit code is 0
+   And Transaction is following
+       | Action        | Package                             |
+       | install       | Package-install-file-0:1.0-1.x86_64 |
+   And stdout contains "triggerin scriptlet \(Package-triggerin-ok\) for Package-install-file install/update successfully done"
+   And stdout does not contain "Running scriptlet\s*:\s*Package-install-file"
+   And stdout contains "Running scriptlet\s*:\s*Package-triggerin-ok"
```

Comment:
Unless there's something nondeterministic in the output, could you use `stdout is`/`stderr is` and an exact match?

Reply:
I probably could but are you sure it's worth it? It would be a lot of unnecessary text in the scenario since I would have to include the whole transaction table and so on. I like that this approach makes it clear what we are checking.

Reply:
Oh, right, there's the transaction as well. Yeah, leave it like this, you'd have to repeat the transaction content and that is counter-productive...

I've got some ideas how we could solve this, but they'll require some time to make reality... :slightly_smiling_face: